File: 1.Arrays/Java/PrintArray.java
import java.util.Scanner;

public class PrintArray {

    public static void main(String args[]){

        int[] arr = new int[5];
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter file values: ");
        for(int i = 0; i<arr.length; i++){
            arr[i] = sc.nextInt();
        }
        for(int i = 0; i<arr.length; i++){
            System.out.print(arr[i]+" ");
        }
        System.out.println();
        sc.close();
    }
}

File: 1.Arrays/Kotlin/PrintArray.kt


import java.util.Scanner;
fun main() {
    var arr = Array(5) { 0 }
    var sc = Scanner(System.`in`)
    print("Enter five values: ")

    for (i in 0..arr.size-1) {
        arr[i] = sc.nextInt()
    }

    for (i in arr) {
        print("$i ")
    }
    println()
}